components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/pom.xml

```diff
@@ -157,7 +162,7 @@
                                         <limit implementation="org.jacoco.report.check.Limit">
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-<!--                                            <minimum>0.50</minimum>-->
+                                            <!--<minimum>0.50</minimum>-->
```

Comment:
why was this commented?

Reply:
The unit test coverage is about 0.28. Is 0.5 a strict necessity?